openapi/external.yaml

```diff
@@ -1,53 +1,491 @@
+              type: integer
+              enum:
+                - 400
+              example: 400
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
+    Unauthorized:
+      allOf:
+        - $ref: '#/components/schemas/ProblemJson'
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum:
+                - 401
+              example: 401
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
     AuthBearer:
-      type: string
-      pattern: ^Bearer [a-zA-Z0-9-_].+
-      description: A JWT auth custom header as `Bearer <JWT Token>`
-      example: >-
-        Bearer
-        eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
-        AuthBearer:
+        type: string
+        pattern: ^Bearer [a-zA-Z0-9-_].+
+        description: A JWT auth custom header as `Bearer <JWT Token>`
+        example: >-
+          Bearer
+          eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+    ApplicationInfo:
+      type: object
+      properties:
+        message:
+          type: string
+      required:
+        - message
     ServiceStatus:
       type: object
       properties:
         message:
           type: string
       required:
         - message
-
+    LockSessionData:
+      type: object
+      properties:
+        unlockCode:
+          type: string
+          minLength: 9
+      required:
+        - unlock_code
+    ProfileData:
+      description: |-
+        Describes the citizen's profile.
+      type: object
+      properties:
+        email:
+          type: string
+          format: email
+          example: foobar@example.com
+        family_name:
+          type: string
+          example: Rossi
+        name:
+          type: string
+          example: Mario
+      required:
+        - name
+        - family_name
+    SessionInfo:
+      description: Info about an user session
+      oneOf:
+        - type: object
+          description: Inactive session
+          properties:
+            active:
+              type: boolean
+              enum:
+                - false
+          required:
+            - active
+        - type: object
+          description: Active session
+          properties:
+            active:
+              type: boolean
+              enum:
+                - true
+            expiration_date:
+              type: string
+              format: date
+          required:
+            - active
+            - expiration_date
+    SessionState:
```

Comment:
We can make a reference to fn-fast-login `SessionState` https://github.com/pagopa/io-functions-fast-login/blob/9c482ce1fcca90ec523bcdfeee00dec2d8bcd199/api/internal.yaml#L487C5-L487C17